backend/src/models/User.js

```diff
@@ -104,4 +104,8 @@ module.exports = {
     target: 'Location',
     direction: 'out',
   },
+  allowEmbedIframes: {
+    type: 'boolean',
```

Comment:
You could even think about using a `date` here. The `false` would be if it's null.

Reply:
I like that suggestion... maybe it's useful to know at some point when the user opted in to this "feature"

Reply:
Although, it would have to be the `String` representation of a `datetime`, no @roschaefer?